models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -199,7 +199,7 @@ public function getFieldType(): string
 
     public function getClasses(): array
     {
-        if($this->ownerClassId) {
+        if(!empty($this->ownerClassName)) {
```

Comment:
empty() is not needed. Attribute is always set.
```suggestion
        if($this->ownerClassName) {
```

Reply:
I now added your suggested change. Also I think the $ownerClassId logic needs to be reevaluated. As far as I saw, it is only set in getOwnerClassId() and that if certain if conditions are met.